content/projects/gsoc/2023/project-ideas/docker-compose-build.adoc

```diff
@@ -1,10 +1,10 @@
 ---
 layout: gsocprojectidea
-title: "Building Jenkins.io with docker-compose"
+title: "Jenkins docker examples to help users get started"
```

Comment:
IMHO, the title should be reworked to be more impactful.
How about "Docker-based Jenkins quickstart examples" ?

Reply:
Sure thing.